modules/private/hook.go

```diff
@@ -31,6 +32,7 @@ type GitPushOptions map[string]string
 const (
 	GitPushOptionRepoPrivate  = "repo.private"
 	GitPushOptionRepoTemplate = "repo.template"
+	GitPushOptionMergePulls   = "merge-pulls"
```

Comment:
Would it make more sense to name it `pulls.merge` to keep using the same naming convention?